lib/Flux.js

```diff
@@ -65,20 +99,39 @@ class Flux {
     stores.map((store) => this.store(store));
   }
 
-  loadState(initialState) {
-    this.stores.forEach(([, store], k) => store.loadState(initialState[k]));
+  /**
+   * Loads each stores of the Flux with the data from the provided Flux state
+   * @param {Array<Object>} state Flux's stores states
+   * @return {Flux} Flux instance
+   */
+  loadState(state) {
+    this.stores.forEach(([, store], k) => store.loadState(state[k]));
     return this;
   }
 
+  /**
+   * Generates an array with each Flux's stores states.
+   * @return {Array<Object>} [description]
+   */
   dumpState() {
     return this.stores.map(([, store]) => store.dumpState());
   }
 
+  /**
+   * Adds a new Action in the FLux.
+   * @param {Action} action Action to add
+   * @return {Action} Added action
+   */
   addAction(action) {
     this.actions.push([createMatcher(action), action]);
     return action;
   }
 
+  /**
+   * Find an action given a route.
+   * @param {String} needle Route of the action to find
+   * @return {Action} First Action matching the route
+   */
   findAction(needle) {
```

Comment:
So you have to change the body of the function in order to avoid `route` === `route` ^.^